squire_lib/src/tournament.rs

```diff
@@ -1112,6 +1112,11 @@ impl TournamentSeed {
             format,
         })
     }
+
+    /// Validates a tournament name. Returns true if the name is valid.
+    pub fn validate_name(name: &String) -> bool {
```

Comment:
nit: You generally don't want to pass `&String` since they can be trivially converted into `&str` via `Deref` coercion.
```suggestion
    pub fn validate_name(name: &str) -> bool {
```